tests/console/yast2_proxy.pm

```diff
@@ -83,19 +83,23 @@ sub run {
         if (check_screen 'yast2_proxy_firewall_enabled') {
             send_key 'alt-p';
             wait_still_screen 3;
+            # now we have a new popup to workaround for question about firewall settings
+            if (check_screen 'yast2_proxy_squid_cannot-open-interface') {
+                send_key 'alt-y';
+            }
         }
     }
 
     # check network interfaces with open port in firewall
     # repeat action as sometimes keys are not triggering action on leap if workers are slow
-    send_key_until_needlematch 'yast2_proxy_network_interfaces', 'alt-d', 2, 5;
-
-    wait_still_screen 1;
-    send_key 'alt-n';
-    wait_still_screen 1;
-    send_key 'alt-a';
-    wait_screen_change { send_key 'alt-o' };
-
+    if ((is_tumbleweed) || (is_sle && !sle_version_at_least('15'))) {
```

Comment:
What's about leap? I believe we need even more ugly check here with "|| !leap_version_at_least('15')"

Reply:
leap 15.0 has been covered now: http://e13.suse.de/tests/4570#step/yast2_samba/32

Reply:
Please try to avoid the explicit statement of `is_tumbleweed` as it should be considered our reference which we never need to check for. Rather turn the check around. Also you are not yet stating the Leap changes. Please make sure your code is always pushed in the most recent version before commenting back that it has been solved so that we know what to check for. On top I don't understand why (new) tumbleweed should behave the same as old-SLE? Is there a package submission gone missing for openSUSE:Factory?

Reply:
this comment still applies

Reply:
changed now.